package.json

```diff
@@ -6,9 +6,13 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
+    "prop-types": "^15.8.1",
```

Comment:
- [ ] [OPTIONAL] Please kindly use the README template from Microverse next time, you can get it [from here](https://github.com/microverseinc/readme-template/blob/master/README.md)